src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -66,21 +71,36 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
                                 title={translate('workspace.qbo.defaultVendor')}
                                 wrapperStyle={[styles.ph5, styles.mb3, styles.mt1]}
                                 isActive={Boolean(autoCreateVendor)}
-                                onToggle={(isOn) => Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR, isOn)}
+                                onCloseError={() => Policy.clearQBOErrorField(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR)}
+                                onToggle={(isOn) => {
+                                    if (!isOn) {
+                                        Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.EXPORT_COMPANY_CARD_ACCOUNT);
+                                    } else {
+                                        Connections.updatePolicyConnectionConfig(
+                                            policyID,
+                                            CONST.POLICY.CONNECTIONS.NAME.QBO,
+                                            CONST.QUICK_BOOKS_CONFIG.EXPORT_COMPANY_CARD_ACCOUNT,
+                                            vendors?.[0]?.name,
+                                        );
+                                    }
+                                    Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR, isOn);
```

Comment:
This breaks the `1:1:1` API design rule (only one API request per user action).

Reply:
@hayata-suenaga @aldo-expensify can you guys confirm this is being worked on in [this issue](https://github.com/Expensify/Expensify/issues/390731)?

Reply:
yes, this is one of the bullet points in [the linked PR](https://github.com/Expensify/Expensify/issues/390731)

> When the admin changes the type of QBO account to export company card (non-reimbursable) expenses to, don't call two updatePolicyConnectionConfig twice to update exportCompanyCard and exportCompanyCardAccount fields. Instead, create a single Web Expensify command to update the both fields at once.